ds-controller-board/res/configuration.toml

```diff
@@ -85,4 +78,12 @@ PublishTopicPrefix = "edgex/events/device" # /<device-profile-name>/<device-name
   Retained = "false"
   AutoReconnect = "true"
   ConnectTimeout = "5" # Seconds
-  SkipCertVerify = "false"
\ No newline at end of file
+  SkipCertVerify = "false"
+
+[AppCustom]
+  [DriverConfig]
+  VID = '2341'
+  PID = '8037'
```

Comment:
Use `"` instead o `'` here and any other places. `"` are the standard. `'` are use in special cases when the string is wrapped across lines.